login-workflow/docs/existing-project-migration.md

```diff
@@ -0,0 +1,135 @@
+
+## Update RegistrationUIActions
+
+Update type name `AccountDetailInformation` to `AccountDetails`.
+
+// before
+```tsx
+import { RegistrationUIActions, AccountDetailInformation } from '@brightlayer-ui/react-auth-workflow';
+```
+
+// after
+```tsx
+import { RegistrationUIActions, AccountDetails } from '@brightlayer-ui/react-auth-workflow';
+```
+
+### Fix Following Actions in RegistrationUIActions.tsx
+
+#### Rename `loadEULA` action to `loadEula`.
+// before
+```tsx
+loadEULA: async (language: string): Promise<string> => {
+    await sleep(1000);
+
+    if (isRandomFailure()) {
+        throw new Error('Sorry, there was a problem sending your request.');
+    }
+
+    if (!language.includes('en')) {
+        return 'Other language EULA';
+    }
+
+    return SAMPLE_EULA;
+},
+```
+
+// after
+```tsx
+loadEula: async (language: string): Promise<string> => {
+    await sleep(1000);
+
+    if (isRandomFailure()) {
+        throw new Error('Sorry, there was a problem sending your request.');
+    }
+
+    if (!language.includes('en')) {
+        return 'Other language EULA';
+    }
+
+    return SAMPLE_EULA;
+}
+```
+
+#### Update completeRegistration Action
+
+// Before
+```tsx
+completeRegistration: async (
+    userData: {
+        password: string;
+        accountDetails: AccountDetailInformation;
+    },
+    validationCode: string,
+    validationEmail?: string
+): Promise<{ email: string; organizationName: string }> => {
+    const email = 'example@email.com';
+    const organizationName = 'Acme Co.';
+    const userInfo = { email, organizationName };
+
+    await sleep(1000);
+    if (isRandomFailure()) {
+        throw new Error('Sorry, there was a problem sending your request.');
+    }
+    return userInfo;
+},
+```
+
+// After
+```tsx
+completeRegistration: async (
+    userData: any,
+    validationCode: string | number,
+    validationEmail?: string
+): Promise<{ email: string; organizationName: string }> => {
+    const email = 'example@email.com';
+    const organizationName = 'Acme Co.';
+    const userInfo = { email, organizationName };
+
+    await sleep(1000);
+    if (isRandomFailure()) {
+        throw new Error('Sorry, there was a problem sending your request.');
+    }
+    return userInfo;
+},
+```
+
+#### Add new actions
+
+We have added few more actions in registraionUIActions such as `acceptEula`, `requestRegistrationCode`, `createPassword`, and `setAccountDetails`.
+For more details you can check implementation of Example app's [RegistraionUIActions Implementation](https://github.com/etn-ccis/blui-react-workflows/blob/master/login-workflow/example/src/actions/RegistrationUIActions.tsx).
+
+
+### Update you App.tsx
```

Comment:
```suggestion
### Update your App.tsx
```

Reply:
I want more handholding here. I'm currently trying to migrate an old project and am getting hung up here. I could surely compare and contract the example project on my own, but it may be a good idea to link out to appropriate files. I would also mention the removal of the AuthUIContextProvider first.

Reply:
Hi @daileytj , I have updated steps. Please review